### DISCUSSION TITLE
Add roblox provider

### DISCUSSION THREAD

**Discussion by marcusleonas** [nextauthjs/next-auth#8732] [Ideas]
### Goals

Add roblox as a provider option

### Non-Goals

1.
2.
3.

### Background

Would make it easier for people to make roblox product hubs & tools.

### Proposal

Same as other providers.

**Comment by marcusleonas** [comment]
Is anyone going to reply to this?

**Reply by ghost** [reply]
Lol they are probably busy, but I would love to see roblox OAuth added to this!
